Neos.ContentRepository.BehavioralTests/Tests/Behavior/Features/ContentStreamForking/01-ForkContentStream_ConstraintChecks.feature

```diff
@@ -49,12 +49,13 @@ Feature: ForkContentStream Without Dimensions
       | propertyValues               | {"text": {"value": "original value", "type": "string"}} |
       | propertiesToUnset            | {}                                                      |
 
-  Scenario: Try to fork a content stream that is closed:
+  Scenario: Try to fork a content stream (use it as base workspace) that is closed:
```

Comment:
I don't understand this completely, is it:
```suggestion
  Scenario: Try to create a workspace with the base workspace referring to a closed content stream
```
?

Reply:
that is correct